lib/rules/property-no-vendor-prefix/index.js

```diff
@@ -11,15 +14,33 @@ const messages = ruleMessages(ruleName, {
   rejected: property => `Unexpected vendor-prefix "${property}"`
 });
 
-const rule = function(actual) {
+const rule = function(actual, options) {
   return (root, result) => {
-    const validOptions = validateOptions(result, ruleName, { actual });
+    const validOptions = validateOptions(
+      result,
+      ruleName,
+      { actual },
+      {
+        optional: true,
+        actual: options,
+        possible: {
+          ignoreProperties: [_.isString]
+        }
+      }
+    );
+
     if (!validOptions) {
       return;
     }
 
     root.walkDecls(decl => {
-      const prop = decl.prop;
+      const prop = decl.prop,
+        unprefixedProp = postcss.vendor.unprefixed(prop);
```

Comment:
Use one `const` per assignment, please. We should've put a rule about this in our ESLint config.